packages/mdc-checkbox/_mixins.scss

```diff
@@ -65,7 +65,6 @@ $mdc-checkbox-ripple-target: ".mdc-checkbox__ripple";
 
   @include mdc-checkbox-container-colors($query: $query);
   @include mdc-checkbox-ink-color($mdc-checkbox-mark-color, $query);
-  @include mdc-checkbox-touch-target($mdc-checkbox-ripple-size, $query);
```

Comment:
Should this be removed?

Reply:
This is indirectly called via density mixin (`density` => `ripple-size` => `touch-target`) making this a duplicate.